pkg/controllers/managementuser/secret/secret.go

```diff
@@ -108,8 +109,8 @@ func registerDeferred(ctx context.Context, cluster *config.UserContext) {
 		}
 
 		if obj.Labels != nil {
-			if obj.Labels["cattle.io/creator"] == "norman" {
-				logrus.Tracef("secretsController: AddHandler: obj [%s] labels in [%s] contain cattle.io/creator=norman, calling sync", obj.Name, cluster.ClusterName)
+			if obj.Labels["cattle.io/creator"] == "norman" || obj.Labels[projectScopedLabel] == "original" {
+				logrus.Tracef("secretsController: AddHandler: obj [%s] in cluster [%s] has a label for a project-scoped secret, calling sync", obj.Name, cluster.ClusterName)
```

Comment:
Might be worth re-wording this to be more specific to the label. If I was looking for a `project-scoped secret`, I'm not sure `cattle.io/creator=norman` would be an indicator of that.